components/Dialogs/src/MDCDialogTransitionController.m

```diff
@@ -18,13 +18,24 @@
 
 @implementation MDCDialogTransitionController
 
-static const NSTimeInterval MDCDialogTransitionDuration = 0.27;
+static const NSTimeInterval MDCDialogDefaultTransitionDuration = 0.27;
+
+static const CGFloat MDCDialogDefaultScaleFactor = 1.0f;
+
+- (instancetype)init {
+  self = [super init];
+  if (self) {
+    self.animationDuration = MDCDialogDefaultTransitionDuration;
```

Comment:
Please set the ivars directly and [avoid referencing `self` inside the initializer](http://google.github.io/styleguide/objcguide.html#avoid-messaging-the-current-object-within-initializers-and--dealloc).

```objc
if (self) {
  _animationDuration = kDefaultTransitionDuration;
  _dialogScaleFactor = kDefaultScaleFactor;
}
```

Reply:
Done.